ambry-clustermap/src/test/java/com.github.ambry.clustermap/MockClusterAgentsFactory.java

```diff
@@ -57,8 +57,8 @@ public void close() {
         }
 
         @Override
-        public void setReplicaSealedState(ReplicaId replicaId, boolean isSealed) {
-
+        public boolean setReplicaSealedState(ReplicaId replicaId, boolean isSealed) {
+          return true;
```

Comment:
shouldn't the state be set in the `MockReplicaId` here? Same as the helix version, this needs to check if `ReplicaId` is an instance of `MockReplicaId`, cast and set I think